herbert_core/admin/@opt_config_manager/private/find_comp_type_.m

```diff
@@ -24,14 +24,18 @@
             pc_type = types{1};%windows small
         end
     end
-
+    if is_jenkins()
```

Comment:
this `if` should be the first check inside the `if ispc` block as the other type detection is unnecessary:
```
if is_jenkins()
    pc_type = types{8};  % 'jenkins_win'
else
    [~,sys] = memory();
    etc...
end
```

Reply:
this function also returns memory size and number of processors. This is why the check is last.